src/pages/index.tsx

```diff
@@ -69,7 +75,36 @@ export default function Page() {
           on AWS. Get started by selecting your preferred framework.
         </Text>
       </Flex>
-
+      <LinkCardCollection>
+        <LinkCard
+          isExternal={true}
+          href={''}
+          icon={() => <IconGithub fontSize="2rem" />}
+        >
+          {'JavaScript Libraries on Github'}
```

Comment:
Don't these need to be variable based on the platform selected?

Reply:
@hbuchel   , please let me know!